fleet-packages-quickstart/README.md

```diff
@@ -0,0 +1,236 @@
+# Multi-cluster configuration with Fleet Packages
+
+[Fleet Packages] provides a way to deploy configuration into multiple clusters in a fleet.
```

Comment:
Is `fleet` term expected to be known by the target customers ?. Or may be we can qualify it with`GKE fleet` to make it more explicit.

Reply:
IIRC, GKE fleet is not the recommended term since a fleet could be fully non-GKE

Reply:
I added a link to the fleet documentation here, so I think that should give the necessary context for anyone who doesn't know what fleet is about.